### ISSUE TITLE
README for new contributors

### ISSUE THREAD

**Issue by zelvik** [GetSimpleCMS/GS_LANGS#3]
Hi,

what is state with this repository? I think, that it could be good, if it was created a readme for new contributors a people from different countries could update continuously their translates. I'm not sure, how stable is actual system placing this translates to plugins, but I'm experienced, that this translates with new versions often doesn't work, so I hope, that one centralized place like github could improve it. The main benefit is, that each person can view what is different from previous version and each person can update it :). What do you think about it?

**Comment by tablatronix** [comment]
Nobody started using it.
I plan on using it to pull in user langs into core and keep them organized and easily generate diffs.

Langs are still user contributed

**Comment by zelvik** [comment]
Ok, it sounds great. And what if we will create some "readme" for new contributors? I think document which will be on main page of this repository, there it will be basic about repository and something about possibilities how people can contribute?
I think, that could be good first step, after I could create, or extend, actual czech translation.
And I hope, that next people help us, if we will post it on get-simple forum. What do you think? I'm relatively newbie in GS (I'm using it a few months), so I'm not sure how much it is possible, but if you think that it is possible, I would like help with it :)

**Comment by tablatronix** [comment]
yeah i have a forum post, i meant to create the readme and contribute files.

**Comment by tablatronix** [comment]
http://get-simple.info/forums/showthread.php?tid=4160

**Comment by zelvik** [comment]
Ok, I updated in plugins now and eventually we can put it here too :)

**Comment by tablatronix** [comment]
Updated what? Not sure what you mean

**Comment by zelvik** [comment]
I updated language plugin (czech), if you want, you can see it here http://get-simple.info/extend/language/czech-language/987/

**Comment by tablatronix** [comment]
Is this new or did you copy someones language, you should attributenor consult with the author if you have changes

**Comment by tablatronix** [comment]
Is this new or did you copy someones language, you should attribute consult with the author if you have changes
